src/test/java/com/wikia/webdriver/testcases/mobilewikitests/curatedcontenttests/MainPageTests.java

```diff
@@ -311,141 +135,68 @@ public void MercuryCuratedMainPageTest_004_CheckElementsVisibilityExceptTrending
   }
 
+
   @Test(groups = "MercuryCuratedMainPageTest_005")
   @Execute(onWikia = MercuryWikis.MERCURY_NTAV_CC)
   public void MercuryCuratedMainPageTest_005_CheckElementsVisibilityExceptTrendingVideosAndArticles() {
     init();
-
     navigate.toPageByPath(MercurySubpages.NTAVCC_MAIN_PAGE);
     Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MOBILE_WIKI));
 
-    boolean result = curatedMainPage.isMobileTopLeaderboardVisible();
-    PageObjectLogging.log(
-        PageElements.TOP_LEADERBOARD.name,
-        MercuryMessages.VISIBLE_MSG,
-        MercuryMessages.INVISIBLE_MSG,
-        result
+    List<String> elemntsWhiuchShouldBeDisplayed = Arrays.asList(
+            AD_TOP_LEADERBOARD_SELECTOR,
+            ARTICLE_TITLE_SELECTOR,
+            WIKIA_STATS_SELECTOR,
+            FEATURED_CONTENT_SELECTOR,
+            CURATED_CONTENT_SELECTOR,
+            AD_MOBILE_IN_CONTENT_SELECTOR
     );
 
-    result = curatedMainPage.isArticleTitleVisible();
-    PageObjectLogging.log(
-        PageElements.ARTICLE_TITLE.name,
-        MercuryMessages.VISIBLE_MSG,
-        MercuryMessages.INVISIBLE_MSG,
-        result
+    List<String> elemntsWhiuchShouldNotBeDisplayed = Arrays.asList(
+            TRENDING_ARTICLES_SELECTOR,
+            TRENDING_VIDEOS_SELECTOR
     );
 
-    result = curatedMainPage.isWikiaStatsContainerVisible();
-    PageObjectLogging.log(
-        PageElements.WIKIA_STATS.name,
-        MercuryMessages.VISIBLE_MSG,
-        MercuryMessages.INVISIBLE_MSG,
-        result
-    );
-
-    result = curatedMainPage.isFeaturedContentVisible();
-    PageObjectLogging.log(
-        PageElements.FEATURED_CONTENT.name,
-        MercuryMessages.VISIBLE_MSG,
-        MercuryMessages.INVISIBLE_MSG,
-        result
-    );
-
-    result = curatedMainPage.isCuratedContentVisible();
-    PageObjectLogging.log(
-        PageElements.CURATED_CONTENT.name,
-        MercuryMessages.VISIBLE_MSG,
-        MercuryMessages.INVISIBLE_MSG,
-        result
-    );
+    verifyElementsVisible(elemntsWhiuchShouldBeDisplayed);
+    verifyElementsNotVisible(elemntsWhiuchShouldNotBeDisplayed);
+  }
 
-    result = curatedMainPage.isMobileInContentVisible();
-    PageObjectLogging.log(
-        PageElements.IN_CONTENT.name,
-        MercuryMessages.VISIBLE_MSG,
-        MercuryMessages.INVISIBLE_MSG,
-        result
+  private void verifyElementsVisible(List<String> elementsList){
+    elementsList.forEach((element) -> Assertion.assertTrue(curatedMainPage.isCuratedElementVisible(element),
+            element + " is not visible")
     );
+  }
 
-    result = !curatedMainPage.isTrendingArticlesVisible();
-    PageObjectLogging.log(
-        PageElements.TRENDING_ARTICLES.name,
-        MercuryMessages.INVISIBLE_MSG,
-        MercuryMessages.VISIBLE_MSG,
-        result
+  private void verifyElementsNotVisible(List<String> elementsList){
+    elementsList.forEach((element) -> Assertion.assertFalse(curatedMainPage.isCuratedElementVisible(element),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "element" parameter (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1611)